packages/terra-slide-group/src/terra-dev-site/test/slide-group/DefaultOneItemSlideGroup.test.scss

```diff
@@ -0,0 +1,5 @@
+:local {
```

Comment:
Do we need two ([AnimatedNonAnimatedSlideGroup.test.scss](https://github.com/cerner/terra-framework/blob/2fcbf0eb8727d4d15b8c0da41131181555c53c06/packages/terra-slide-group/src/terra-dev-site/test/slide-group/AnimatedNonAnimatedSlideGroup.test.scss) and [DefaultOneItemSlideGroup.test.scss](https://github.com/cerner/terra-framework/blob/2fcbf0eb8727d4d15b8c0da41131181555c53c06/packages/terra-slide-group/src/terra-dev-site/test/slide-group/DefaultOneItemSlideGroup.test.scss))stylesheets for slide-group tests OR can we just have single scss file for slide group having all these classes defined within it.

Reply:
fixed here https://github.com/cerner/terra-framework/pull/692/commits/6c706bab37454c6d9c4fb8a8ee2678b5ccc57890.